packages/core/src/stryker-cli.ts

```diff
@@ -149,6 +149,10 @@ export class StrykerCli {
         `Set the log4js log level for the "stryker.log" file. Possible values: fatal, error, warn, info, debug, trace and off. Default is "${defaultOptions.fileLogLevel}"`,
       )
       .option('--allowConsoleColors <true/false>', 'Indicates whether or not Stryker should use colors in console.', parseBoolean)
+      .option(
+        '--useMutationLevel <mutationLevel>',
+        'Pick a mutation level to be selected during a mutation run, which will restrict the types of mutator operators which will be executed. The selected mutation level must be defined in the configuration file, except for the three default ones: X,Y,Z.',
```

Comment:
I'm not sure what you mean here, are X, Y and Z placeholders?

Reply:
Yes, because we don't have default levels yet.

Reply:
Might be good to add a TODO there in that case, or to add an issue that we should set default levels. I'm a bit worried we might forget to change this in the future otherwise